website/docs/reference/api/unleash/sidebar.js

```diff
@@ -0,0 +1 @@
label":"getTagsByType","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/get-tag","label":"getTag","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/delete-tag","label":"deleteTag","className":"api-method delete"},{"type":"doc","id":"reference/api/unleash/get-tag-types","label":"getTagTypes","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/create-tag-type","label":"createTagType","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/validate-tag-type","label":"validateTagType","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/get-tag-type","label":"getTagType","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/update-tag-type","label":"updateTagType","className":"api-method put"},{"type":"doc","id":"reference/api/unleash/delete-tag-type","label":"deleteTagType","className":"api-method delete"}]},{"type":"category","label":"Unstable","link":{"type":"doc","id":"reference/api/unleash/unstable"},"items":[{"type":"doc","id":"reference/api/unleash/get-frontend-features","label":"getFrontendFeatures","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/register-frontend-metrics","label":"registerFrontendMetrics","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/register-frontend-client","label":"registerFrontendClient","className":"api-method post"}]},{"type":"category","label":"Users","link":{"type":"doc","id":"reference/api/unleash/users"},"items":[{"type":"doc","id":"reference/api/unleash/get-me","label":"getMe","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/get-profile","label":"getProfile","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/change-my-password","label":"changeMyPassword","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/validate-user-password","label":"validateUserPassword","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/change-user-password","label":"changeUserPassword","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/reset-user-password","label":"resetUserPassword","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/get-users","label":"getUsers","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/create-user","label":"createUser","className":"api-method post"},{"type":"doc","id":"reference/api/unleash/search-users","label":"searchUsers","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/get-base-users-and-groups","label":"getBaseUsersAndGroups","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/get-user","label":"getUser","className":"api-method get"},{"type":"doc","id":"reference/api/unleash/update-user","label":"updateUser","className":"api-method put"},{"type":"doc","id":"reference/api/unleash/delete-user","label":"deleteUser","className":"api-method delete"}]}];
```

Comment:
This file should not be included in git. Could you remove it?

Reply:
Removed it again, also added it to the .gitignore this time, turns out its not actually in main

Reply:
Huh, thought that was fixed in https://github.com/Unleash/unleash/pull/2070/files 🤔

Reply:
That nested .gitignore seems to be completely ignored by my git for some reason. Not sure why but adding it to the base .gitignore seems to sort it out